utils/config.py

```diff
@@ -1,32 +1,29 @@
 import os
 import sys
 from brownie import network, accounts
+from brownie.convert.datatypes import Wei
 
 ldo_token_address = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
 dai_address = "0x6b175474e89094c44da98b954eedeac495271d0f"
 steth_token_address = "0xae7ab96520de3a18e5e111b5eaab095312d7fe84"
+steth_token_kernel = "0xb8ffc3cd6e7cf5a098a1c92f48009765b24088dc"
 one_inch_token_address = "0x111111111117dc0aa78b770fa6a738034120c302"
 one_inch_token_owner = "0x5E89f8d81C74E311458277EA1Be3d3247c7cd7D1"
 lido_dao_agent_address = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
-steth_token_kernel = "0xb8ffc3cd6e7cf5a098a1c92f48009765b24088dc"
-lp_token_address = "0xc5578194d457dcce3f272538d1ad52c68d1ce849"
-
-
-'''
-lido_dao_token_manager_address = "0xf73a1260d222f447210581DDf212D915c09a3249"
-wsteth_address = "0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0"
-lido_dao_finance_address = "0xB9E5CBB9CA5b0d659238807E84D0176930753d86"
 lido_dao_voting_address = "0x2e59A20f205bB85a89C53f1936454680651E618e"
-'''
+lp_token_address = "0xc5578194d457dcce3f272538d1ad52c68d1ce849"
```

Comment:
Seems like this variable isn't used across the project as a couple of others. It's better to remove them.

Reply:
Addressed in https://github.com/maddevsio/lido/pull/3/commits/f7dda2a35b326c5a527d155dd09dd5d01e08c905